highfidelity/hifi
# Fix tablet home button highlight overlay dimensions

[FB9368](https://highfidelity.fogbugz.com/f/cases/9368)

The highlight overlay was not handling its dimensions properly.  The radii automatically scale with dimensions, so only the dimension needed to be changed, but circle overlays currently render at the wrong size ([FB9365](https://highfidelity.fogbugz.com/f/cases/9365)), so the dimensions were off.

Test plan:
- Hover over the tablet home button with the lasers or stylus.  It should highlight properly.
- Change size.  The size of the home button highlight should still be correct.  (The position might be slightly off sometimes, that is a separate bug)
